components/dashboards-web-component/src/designer/DashboardDesigner.css

```diff
@@ -96,4 +96,46 @@
 
 .home-page-checkbox {
     padding: 30px 0px;
-}
\ No newline at end of file
+}
+
+.designer-left-menu {
+    padding: 0 !important;
+}
+
+/* dashboard designer golden layout style customizations */
+.dashboard-designer .lm_header.lm_selectable {
+    background: #192830 !important;
+    height: 37px !important;;
+}
+
+.dashboard-designer .lm_content {
+    background: #13222a !important;
+    border-color: #13222a !important;
+}
+
+.dashboard-designer .lm_header .lm_tab.lm_active {
+    box-shadow: none;
+    background: #13222a;
+    height: 25px;
+    margin-top: 0px;
+    border: none;
```

Comment:
When the value is 0 no need to specify the unit, pls check and fix in other places as well.

Reply:
fixed